src/compiler/diagnosticMessages.json

```diff
@@ -4358,6 +4358,10 @@
         "category": "Message",
         "code": 6383
     },
+    "Skip checking and generating .d.ts files for files indirectly importing affected file": {

```

Comment:
This only affects `--build`, right? Does it also only affect `--watch`?

Reply:
It does.. It affects incremental scenarios. (it does not type check and does not emit .d.ts for files that are indirectly importing file with changed signature)